test/hotspot/jtreg/serviceability/jvmti/StartPhase/AllowedFunctions/AllowedFunctions.java

```diff
@@ -23,12 +23,13 @@
 
 /**
  * @test
- * @bug 8172970
+ * @bug 8172970 8362203
  * @summary Verify the functions that are allowed to operate in the start phase
  * with and without can_generate_early_vmstart capability.
  * @requires vm.jvmti
  * @run main/othervm/native -agentlib:AllowedFunctions AllowedFunctions
  * @run main/othervm/native -agentlib:AllowedFunctions=with_early_vmstart AllowedFunctions
+ * @run main/othervm/native -agentlib:AllowedFunctions=with_early_vmstart  -Xrunjdwp:transport=dt_socket,address=8000,server=y,suspend=n AllowedFunctions
```

Comment:
Do I understand correctly this test case reproduces the issue?
- there is extra space before `-Xrunjdwp`
- would be better to use automatic port (address=0) to avoid "port in use"
- need to update (c) years in the file

Reply:
Thank, I updated port, indentation and years.